server/service/src/sync/translations/stock_line.rs

```diff
@@ -44,6 +44,7 @@ pub struct LegacyStockLineRow {
     #[serde(deserialize_with = "empty_str_as_option_string", rename = "barcodeID")]
     pub barcode_id: Option<String>,
     #[serde(rename = "om_item_variant_id")]
+    #[serde(default)]
```

Comment:
I think I added this here by accident, but I think it doesn't hurt to allow this? Otherwise potentially could be an compatibility issue with older mSupply?